src/include/migraphx/matcher.hpp

```diff
@@ -307,6 +308,12 @@ struct matcher_result
         {
         }
 
+        instruction_container& operator=(const std::unordered_map<std::string, instruction_ref>& x)
```

Comment:
is this used anywhere ?

Reply:
I rechecked, and it turned out that change is not needed anymore. I have updated the PR.